packages/driver/cypress/integration/e2e/multi_domain_spec.ts

```diff
@@ -136,9 +147,7 @@ describe('multi-domain', { experimentalSessionSupport: true, experimentalMultiDo
           done()
         })
 
-        cy.get('[data-cy="alert"]').then(($el) => {
-          $el.trigger('click')
-        })
+        cy.get('[data-cy="alert"]').click()
```

Comment:
Bonus fix - looks like the nested elements bug was the underlying cause of `click` not working, which I hadn't previously punted on by using `trigger('click')` instead.